lib/Lifters/FunctionLifter.cpp

```diff
@@ -1322,7 +1321,7 @@ llvm::Function *FunctionLifter::GetOrDeclareFunction(const FunctionDecl &decl) {
 void FunctionLifter::AllocateAndInitializeStateStructure(
     llvm::BasicBlock *block) {
   llvm::IRBuilder<> ir(block);
-  const auto state_type = state_ptr_type->getElementType();
+  const auto state_type = llvm::StructType::getTypeByName(ir.getContext(), "struct.State");
```

Comment:
Use the architecture here.

Reply:
Hrmm, though I've asked you to mark the state structure type method as deprecated in remill. I guess undo that.. !